client/packages.go

```diff
@@ -72,3 +72,15 @@ func (client *Client) Packages() (map[string]Package, error) {
 
 	return packages, nil
 }
+
+// Package returns details of a single Package
```

Comment:
Using type names in comments is of course fine. But this is not a type name.. the function does not return details of a Package. It returns a Package value, instead. But we don't need to say that either, because that's obvious from the prototype.

The more interesting ideas to point out in a comment is what's not obvious from the prototype. For example, I would replace the entire comment with:

_// Package returns the most recently published revision of the package with the provided name._

Assuming that's true. If it's not true, what is the actual truth? That's worth reading a comment for.